src/aiida/cmdline/commands/cmd_code.py

```diff
@@ -240,29 +241,35 @@ def show(code):
 @options.OVERWRITE()
 @options.SORT()
 @with_dbenv()
-def export(code, output_file, overwrite, sort):
+def export(code, output_file, overwrite, sort, **kwargs):
```

Comment:
Why are the `kwargs` now necessary? They should always be empty, shouldn't they?